src/SFA.DAS.ApprenticeCommitments.Database/Tables/Apprenticeship.sql

```diff
@@ -3,7 +3,10 @@
 	[Id] BIGINT IDENTITY(10000,1) NOT NULL,
 	[ApprenticeId] UNIQUEIDENTIFIER NOT NULL, 
     [CommitmentsApprenticeshipId] BIGINT NOT NULL,
-	CONSTRAINT PK_Apprenticeship_Id PRIMARY KEY CLUSTERED ([Id]),
+	[EmployerName] NVARCHAR(100) NOT NULL, 
+	[EmployerAccountLegalEntityId] BIGINT NOT NULL,
+    [TrainingProviderName] NVARCHAR(100) NULL, 
```

Comment:
Training provider Name should not be nullable, Also thinking about it. Should we also be saving the ProviderId as this uniquely identifies the provider (same logic as the employer)

Reply:
Good point.  I've added the Provider ID and made everything mandatory.